sample/001_banner/pbjs.html

```diff
@@ -12,8 +12,8 @@
             border-style: solid;
         }
     </style>
-    <script src="//cdn.jsdelivr.net/npm/prebid.js@latest/dist/not-for-prod/prebid.js"></script>
-    <script>
+    <script src="//cdn.jsdelivr.net/npm/prebid.js@latest/dist/not-for-prod/prebid.js" defer></script>
+    <script defer>
```

Comment:
👍 for adding defer to ensure full html is parsed before running the script + ensure the order of execution of scripts